### PR TITLE
:sparkles: feat: added homepage

### PR DESCRIPTION
**Pull request by rahulk31**
Added Homepage for E-commerce project.

### CODE REVIEW

File: css/utility.css [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+}
+
+body {
+  font-family: "Rubik", sans-serif;
+  box-sizing: border-box;
+  max-width: 1500px;
+}
+
+.primary-color {
+  color: var(--site-primary) !important;
```
Comment (j24m) [inline css/utility.css]: !important as the name suggests is used in very rare cases, so maybe you can remove it ?

**Review by j24m — APPROVED** [review]
I see that you have used most of the components from CL , your code is fine except for the use of !important, this must be avoided, UI looks good.

**Review by mehulsatardekar — APPROVED** [review]
Love the UI of menstack. and didn't find any issues, all looks good from my side.

File: css/utility.css (hunk 1 above)
Comment (rahulk31) [reply css/utility.css]: I have removed the important tags as suggested.

**Pull request merged**